x-pack/docs/en/security/configuring-stack-security.asciidoc

```diff
@@ -112,3 +125,78 @@ include::{es-ref-dir}/setup/install/security-files-reference.asciidoc[leveloffse
 Additionally, when you use the enrollment token to connect {kib} to a secured {es} cluster, the HTTP layer CA certificate is retrieved from {es} and stored in the
 {kib} `/data` directory. This file establishes trust between {kib} and the {es}
 Certificate Authority (CA) for the HTTP layer.
+
+[discrete]
+[[stack-skip-auto-configuration]]
+=== Cases when security auto configuration is skipped
+When you start {es} for the first time, we try to automatically configure
+security for you. We run some checks to determine whether this is the first time
+that the node is starting. If any of those checks fail, there's a good
+indication that you <<<<manually-configure-security,manually configured security>>,
+or don't want security to be configured automatically. In these cases, we'll
+respect your existing configuration and skip the auto-configuration process. 
```

Comment:
"we'll respect your existing configuration" -> the node starts up normaly using the existing configuration